internal/app/test-fixtures/test_config.yaml

```diff
@@ -29,8 +29,14 @@ outputs:
 - pubsub:
     project: my-pubsub-project
     topic: my-topic
-kubernetesAPIClient:
-  enabled: true
-  clientMaxQPS: 55
-  resourceAPIVersions:
-    - v1
+inputs:
+  gkeAPI:
+    enabled: true
+  gkeLocal:
+    enabled: false
+    file:
```

Comment:
Don't really like this empty file here... this can brake when we add validation of this input (and we should in fact).
Therefore I propose to add some dummy path here or from test fixtures if really needed

Reply:
removed file: label it as the local input is not even enabled